drivers/src/memory_layout.rs

```diff
@@ -25,25 +25,26 @@ pub const ROM_ORG: u32 = 0x00000000;
 pub const MBOX_ORG: u32 = 0x30000000;
 pub const ICCM_ORG: u32 = 0x40000000;
 pub const DCCM_ORG: u32 = 0x50000000;
-pub const MAN1_ORG: u32 = 0x50000000;
-pub const MAN2_ORG: u32 = 0x50001800;
-pub const FHT_ORG: u32 = 0x50003000;
-pub const LDEVID_TBS_ORG: u32 = 0x50003800;
-pub const FMCALIAS_TBS_ORG: u32 = 0x50003C00;
-pub const RTALIAS_TBS_ORG: u32 = 0x50004000;
-pub const PCR_LOG_ORG: u32 = 0x50004400;
-pub const FUSE_LOG_ORG: u32 = 0x50004800;
-pub const BOOT_STATUS_ORG: u32 = 0x50004BE4;
-pub const CFI_VAL_ORG: u32 = 0x50004BE8;
-pub const CFI_MASK_ORG: u32 = 0x50004BEC;
-pub const CFI_XO_S0_ORG: u32 = 0x50004BF0;
-pub const CFI_XO_S1_ORG: u32 = 0x50004BF4;
-pub const CFI_XO_S2_ORG: u32 = 0x50004BF8;
-pub const CFI_XO_S3_ORG: u32 = 0x50004BFC;
-pub const DATA_ORG: u32 = 0x50004C00;
-pub const STACK_ORG: u32 = 0x5001A000;
-pub const ESTACK_ORG: u32 = 0x5001F800;
-pub const NSTACK_ORG: u32 = 0x5001FC00;
+// Region of ~ 1k bytes between DCCM_ORG and CFI_VAL_ORG is reserved for ROM's DATA_ORG
+pub const CFI_VAL_ORG: u32 = 0x500003E8;
+pub const CFI_MASK_ORG: u32 = 0x500003EC;
+pub const CFI_XO_S0_ORG: u32 = 0x500003F0;
+pub const CFI_XO_S1_ORG: u32 = 0x500003F4;
+pub const CFI_XO_S2_ORG: u32 = 0x500003F8;
+pub const CFI_XO_S3_ORG: u32 = 0x500003FC;
+pub const MAN1_ORG: u32 = 0x50000400;
+pub const MAN2_ORG: u32 = 0x50001C00;
+pub const FHT_ORG: u32 = 0x50003400;
+pub const LDEVID_TBS_ORG: u32 = 0x50003C00;
+pub const FMCALIAS_TBS_ORG: u32 = 0x50004000;
+pub const RTALIAS_TBS_ORG: u32 = 0x50004400;
+pub const PCR_LOG_ORG: u32 = 0x50004800;
+pub const FUSE_LOG_ORG: u32 = 0x50004C00;
+pub const BOOT_STATUS_ORG: u32 = 0x50004FE4;
```

Comment:
BOOT_STATUS_ORG is also modified by the ROM out-of-band and should be moved to the ROM section.

Reply:
If boot_status is set to non-zero, are we guaranteed to not continue booting to FMC? I want to make sure we don't have a use-case for reading/reporting this value in RT.

But also, iirc there are boot status registers, so I suppose we can use those if needed.

Reply:
I don't think it's a problem if any of these memory location are used by the runtime firmware. The important thing is that the ROM can make writes to this section without disturbing the persistent data.

Reply:
Done